include/tvm/relay/op_attr_types.h

```diff
@@ -207,13 +216,137 @@ enum AnyCodegenStrategy {
   kVariableDimensions
 };
 
-/* \brief A runtime representation of shape. */
+/*! \brief A runtime representation of shape. */
 using Shape = Array<IndexExpr>;
 
 using FShapeFunc = runtime::TypedPackedFunc<
   Array<te::Tensor>(const Attrs& attrs,
-                     const Array<te::Tensor>& inputs,
-                     const Array<IndexExpr>& out_ndims)>;
+                    const Array<te::Tensor>& inputs,
+                    const Array<IndexExpr>& out_ndims)>;
+
+/*!
+ * \brief Operator implementation in TVM.
+ */
+class OpImplementNode : public Object {
```

Comment:
OpImplement -> OpImplementation?

Reply:
I agree OpImplementation should be better. But this name is a bit too long. Do you have better idea?

Reply:
Just OpImplNode?

Reply:
Renamed to OpImplementation, and also updated the related APIs